benchmarks/dbscan/print_timers.cpp

```diff
@@ -21,8 +21,12 @@ struct Timer
 {
   using clock_type = std::chrono::high_resolution_clock;
   std::string label;
-  clock_type::time_point tick;
-  clock_type::duration duration;
+  clock_type::time_point tick = {};
+  clock_type::duration duration = {};
+
+  Timer(std::string label)
+      : label(std::move(label))
+  {}
```

Comment:
I rewrote the initialization in a way that seemed sensible to me. I didn't try random things since I don't have a setup to reproduce the problem locally.

Reply:
Time was intended to be an aggregate